frontend/src/scenes/session-recordings/playlist/Playlist.tsx

```diff
@@ -235,13 +237,12 @@ export function Playlist({
                     </div>
                 </div>
                 <div
-                    className={clsx(
-                        'Playlist h-full min-h-96 w-full min-w-96 lg:min-w-[560px] order-first xl:order-none',
-                        {
-                            'Playlist--wide': size !== 'small',
-                            'Playlist--embedded': embedded,
-                        }
-                    )}
+                    className={clsx('Playlist h-full min-h-full w-full min-w-96 lg:min-w-[560px]', {
+                        'Playlist--wide': size !== 'small',
+                        'Playlist--embedded': embedded,
+                        'order-first xl:order-none': !sidebarOpen,
```

Comment:
i imagine this would be annoying on a larger screen where i do have space
we could do this on size of player? like with the size hooks in my UI nits PR

this is the kind of problem the panels https://github.com/PostHog/posthog/issues/26098 is trying to solve